CITATION.cff

```diff
@@ -93,11 +136,6 @@ authors:
   affiliation: Ramona Optics Inc, Durham, North Carolina, USA
   orcid: https://orcid.org/0000-0002-4657-4603
   alias: hmaarrfk
-- given-names: Kyle I. S.
```

Comment:
@jni you cut Kyle here, but I don't see Kyle pasted back anywhere.

Reply:
gdi. Anyway, code review for the win! 😅 Thanks!